.cargo/config.toml

```diff
@@ -1,5 +1,6 @@
 [build]
 # Uncomment the relevant target for your chip here (ESP32, ESP32-S2, ESP32-S3 or ESP32-C3)
+# you may need to change the toolchain in `rust-toolc
```

Comment:
What is `rust-toolc`?

Reply:
meant to be rust toolchain, xtensa chips need a custom toolchain afaik

Reply:
Seems like an unfinished sentence then?